apps/web/src/views/universalFarms/PositionPage.tsx

```diff
@@ -136,92 +142,80 @@ export const PositionPage = () => {
     [v3Positions],
   )
   const pools = usePoolsWithChainId(v3PoolKeys)
-
-  const { poolsMap: v3Pools } = usePoolsByV3Positions(v3Positions)
-  const { data: v2Positions, pending: v2Loading } = useAccountV2LpDetails(filters.selectedNetwork, account)
-  const { poolsMap: v2Pools } = usePoolsByV2Positions(v2Positions)
-  const { data: stablePositions, pending: stableLoading } = useAccountStableLpDetails(filters.selectedNetwork, account)
-  const { poolsMap: stablePools } = usePoolsByStablePositions(stablePositions)
-
-  const checkStatus = useCallback(
-    (pos: PositionDetail, pool: Pool | null) => {
-      switch (positionStatus) {
-        // all
-        case 0:
-          return true
-        // active
-        case 1:
-          return pool && pool.tickCurrent >= pos.tickLower && pool.tickCurrent < pos.tickUpper && pos.liquidity !== 0n
-        // inactive
-        case 2:
-          return pool && (pool.tickCurrent < pos.tickLower || pool.tickCurrent >= pos.tickUpper) && pos.liquidity !== 0n
-        // closed
-        case 3:
-          return pos.liquidity === 0n
-        default:
-          return true
-      }
-    },
-    [positionStatus],
+  const v3PositionsWithStatus = useMemo(
+    () =>
+      v3Positions.map((pos, idx) =>
+        assign(pos, {
+          status: getPoolStatus(pos, pools[idx][1]),
+        }),
+      ),
+    [v3Positions, pools],
   )
 
+  // const { poolsMap: v3Pools } = usePoolsByV3Positions(v3Positions)
+
+  const v3Pools = useMemo(() => ({}), [])
   const filteredV3Positions = useMemo(
     () =>
-      v3Positions.filter(
-        (pos, idx) =>
-          filters.selectedNetwork.includes(pos.chainId) &&
-          selectedPoolTypes.includes(pos.protocol) &&
-          (!filters.selectedTokens?.length ||
-            filters.selectedTokens.some(
+      v3PositionsWithStatus.filter(
+        (pos) =>
+          selectedNetwork.includes(pos.chainId) &&
+          (!selectedTokens?.length ||
+            selectedTokens.some(
               (token) =>
                 token === toTokenValue({ chainId: pos.chainId, address: pos.token0 }) ||
```

Comment:
try use `.equals` from packages/swap-sdk-core/src/token.ts

Reply:
there is just a token address, not token object